### DISCUSSION TITLE
Stop Function that stops the print immediately.

### DISCUSSION THREAD

**Discussion by Talaxin** [mriscoc/Ender3V2S1#780] [Ideas]
When printing with this firmware, whenever there has been an issue and i have needed to stop the print, after confirming that i want to stop the print(which is a good check), i have to wait for is to first finish a set of gcode before it stops and homes the printhead. This waiting period of time is very problematic when i incorrectly level my print bed. If the nozzle is too low, i have to wait for a lot more damage to be done before the printer stops. I am aware that i can manually turn off the printer but it is impractical and inconvenient.

TLDR: Please make the 'Stop' and 'Pause' print options act immediately after the confirmation instead of finishing a set of gcode first.

Thanks :)
